contentcuration/contentcuration/views.py

```diff
@@ -248,6 +250,56 @@ def compress_video_wrapper(file_object):
         low_res_object.save()
         return low_res_object
 
+def create_tiled_image_wrapper(node, files, preset_id):
+    random.shuffle(files)
+    if len(files) >= 4:
+        files = files[:4]
+    elif len(files) >= 1:
+        files = files[:1]
+
+    with tempfile.TemporaryFile(suffix=".{}".format(file_formats.PNG)) as tempf:
+        tempf.close()
+        create_tiled_image(files, tempf.name)
+
+        filename = write_file_to_storage(open(tempf.name, 'rb'), name=tempf.name)
```

Comment:
Can you move the `open` call to a `with` statement? These will leave the file handles lying around, eventually overwhelming the OS.